### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by JuAn-Kang** [clauderic/dnd-kit#722] [Q&A] [answered]
How frequent are the release cycles?

**Comment by clauderic** [comment, marked as answer]
Every merged commit gets deployed to npm under the `next` tag:

```bash
npm install @dnd-kit/core@next @dnd-kit/sortable@next
```

Typically releases are merged quickly if they do not contain breaking changes. If the releases do contain breaking changes, I try to take the opportunity to think about what other changes may also belong in that release to avoid shipping breaking changes too frequently.
